### ISSUE TITLE
Incorrect definition of unit degF

### ISSUE THREAD

**Issue by ThomasEigner** [AnIML/techniques#10]
The unit "degrees Fahrenheit / degF" is defined as

```xml
<!ENTITY degF "<Unit label='&#176;F'><SIUnit factor='1.8' offset='434.07'>K</SIUnit></Unit>">
```
in animl_unit_entities.dtd.

This definition would result in a conversion
```
F = 1.8 * K + 434.07
```
which is incorrect.

Actually, the conversion would be
```
F = 1.8 * K - 459.67
```

resulting in a definition

```xml
<!ENTITY degF "<Unit label='&#176;F'><SIUnit factor='1.8' offset='-459.67'>K</SIUnit></Unit>">
```

Please doublecheck and fix if appropriate.

**Comment by pboba** [comment]
Since we are converting from Kelvin to Fahrenheit, shouldn't the formula be:

F = (K - 273.15) * 1.8 + 32

https://www.nist.gov/pml/weights-and-measures/si-units-temperature

**Comment by ThomasEigner** [comment]
Well I can't find an obvious difference here as your equation and mine are the same:

```
F = (K - 273.15) * 1.8 + 32
F = 1.8 * K - 273.15 * 1.8 + 32
F = 1.8 * K - 459.67
```

I justed used the expanded form so that factor and offset can directly seen in the equation.
Anything I am misssing here?

**Comment by ThomasEigner** [comment]
Fixed by pboba@fbdfbd5a69310bd86256536f0cd3cfa3aa2a2290
